beta_rec/datasets/epinions.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import os
+import pandas as pd
+
+from beta_rec.utils.constants import *
+from beta_rec.datasets.dataset_base import DatasetBase
+
+epinions_url = 'http://www.trustlet.org/datasets/downloaded_epinions/ratings_data.txt.bz2'
```

Comment:
whether it would be better if we store these constants in a separate file?

Reply:
Agree. import * is not a good coding format. I have raised an issue for this.